mlflow/deployments/utils.py

```diff
@@ -15,3 +19,44 @@ def parse_target_uri(target_uri):
             + "Deployment URIs must be of the form 'target' or 'target:/suffix'"
         )
     return parsed.scheme
+
+
+def _is_valid_uri(uri: str) -> bool:
+    """
+    Evaluates the basic structure of a provided gateway uri to determine if the scheme and
+    netloc are provided
+    """
+    if uri == "databricks":
+        return True
```

Comment:
based on usage in https://github.com/mlflow/mlflow/pull/10458/files#diff-3748e5ad5a103faebffe78f181500a0c7087fc318eb5d4b1c3f234c3fa0c2907R62, do we need / should we actually have this logic?

Reply:
Thanks for the catch, removed